RebelAllianceBank/Classes/Admin.cs

```diff
@@ -18,5 +18,414 @@ public Admin(string pNum, string password, string surname, string forename)
             Surname = surname;
             Forename = forename;
         }
+
+        /// <summary>
+        /// Method to create new user
+        /// </summary>
+        public void CreateUser(List<IUser> users)
+        {
+            string userType = "";
+            string password = "";
+
+            string forename = "";
+            string surname = "";
+
+            string personalNum = "";
+            string birthYear = "";
+            string birthMonth = "";
+            string birthDay = "";
+            string lastDigits = "";
+
+            int daysInBirthMonth = 0;
+
+            bool methodRun = true;
+            //Takes in user input about what type of user that is to be created.
+            while (methodRun)
+            {
+                Console.Clear();
+                Console.WriteLine("Vilken typ av användare vill du skapa." +
+                    "\n1. Kund" +
+                    "\n2. Admin" +
+                    "\n3. Gå tillbak till menyn.");
+                string input = Console.ReadLine();
+                bool validInput = false;
+
+                switch (input)
+                {
+                    case "1":
+                        userType = "Kund";
+                        validInput = true;
+                        break;
+                    case "2":
+                        userType = "Admin";
+                        validInput = true;
+                        break;
+                    case "3":
+                        methodRun = false;
+                        break;
+                    default:
+                        break;
+                }
+
+                if (!methodRun)
+                {
+                    break;
+                }
+                else if (validInput)
+                {
+                    break;
+                }
+                else
+                {
+                    Console.WriteLine("Det är inte et giltligt val. Skriv siffran av det val du vill välja." +
+                        " Tryck på valfri tangent tangent för att gå tillbaka till valen.");
+                    Console.ReadKey();
+                    continue;
+                }
+            }
```

Comment:
I think there might be too many ifs. 
It can be refactor to this:
```C#
int menuSelected;
            Console.Clear();
            Console.WriteLine("Vilken typ av användare vill du skapa." +
                "\n1. Kund" +
                "\n2. Admin" +
                "\n3. Gå tillbak till menyn.");
            while (int.TryParse(Console.ReadLine(), out menuSelected))
            {
                Console.WriteLine("Det är inte et giltligt val. Skriv siffran av det val du vill välja.");
            }

            switch (menuSelected)
            {
                case 1:
                    userType = "Kund";
                    break;
                case 2:
                    userType = "Admin";
                    break;
                case 3:
                    return;
            }
```